packages/iris-grid/src/sidebar/visibility-ordering-builder/VisibilityOrderingBuilder.tsx

```diff
@@ -1119,7 +1150,7 @@ class VisibilityOrderingBuilderInner extends PureComponent<
         [...selectedColumns.values()],
         showHiddenColumns
       ),
-    { max: 1000 }
+    { max: 10 }
```

Comment:
This can probably be `max: 2`, for 2 states of showHiddenColumns. The rest of the args aren't likely to change back and forth, if I understand correctly.

Reply:
Ya 2 should be fine. It's possible that undo/redo state could reuse a little more cache, but it shouldn't be as expensive as it was since other things have also been optimized in this PR (like removing a bunch of `findIndex` scans of the items)